sdk/lib/web3.js

```diff
@@ -68,7 +68,7 @@ async function getLogs({ web3, scan, limiter, batchSize, target, topic, topics,
   // assume scattered events can be returned in one page
   // TODO we may need to avoid this assumption
   // and add pagination support at some point
-  const txs = await scan.getTxList({ address: target, startBlock: fromBlock, endBlock: toBlock });
```

Comment:
Maybe instead of changing the getLogs method I should create a getInternalLogs method?

Reply:
I noticed issue #4 maybe related to this change as well.
should txs be a union of getTxList + getTxListInternal?

Reply:
I agree